torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4213,6 +4213,53 @@ def sample_inputs_to_sparse(op_info, device, dtype, requires_grad, **kwargs):
     return (SampleInput(make_arg((S, S)), args=(), output_process_fn_grad=lambda x: x.to_dense()),
             SampleInput(make_arg((S, S)), args=(1,), output_process_fn_grad=lambda x: x.to_dense()),)
 
+def sample_inputs_cross_entropy(op_info, device, dtype, requires_grad, **kwargs):
+    batch_size, num_classes = shape = (2, 3)
+
+    input_shape_and_kwargs: List[Tuple[Tuple[int, ...], Dict[str, Any]]] = [
+        (shape, dict()),
+        ((*shape, 1), dict()),
+        ((*shape, 1, 2), dict()),
+        ((*shape, 1, 2, 3), dict()),
+        (shape, dict(weight=make_tensor((num_classes,), device=device, dtype=dtype))),
+        (shape, dict(ignore_index=1)),
+        (shape, dict(reduction="sum")),
+        (shape, dict(reduction="none")),
+    ]
+
+    sample_inputs = []
+    for (input_shape, kwargs), probabilities_target in itertools.product(input_shape_and_kwargs, (False, True)):
+        # FIXME:
+        if "weight" in kwargs and not probabilities_target:
+            continue
```

Comment:
On CPU this combination is causing gradgradcheck errors. I'm working on a MWE and will open an issue,

Reply:
See #64256.